bitbots_vision/bitbots_vision/vision_modules/color.py

```diff
@@ -253,7 +313,13 @@ def _mask_image(self, image):
         :return np.array: masked image
         """
         hsv_image = cv2.cvtColor(image, cv2.COLOR_BGR2HSV)
-        return cv2.inRange(hsv_image, self._min_vals, self._max_vals)
+
+        if self._zero_crossing:
+            interval_A = cv2.inRange(hsv_image, self._min_vals_A, self._max_vals_A)
+            interval_B = cv2.inRange(hsv_image, self._min_vals_B, self._max_vals_B)
+            return np.clip(interval_A + interval_B, 0, 255)
```

Comment:
We could just do 

```python
return interval_A | interval_B
```

It also works on arrays. It is a binary operator, but the values returned by `cv2.inRange` are `255`, so it doesn't really matter.

Reply:
Good idea, will change that.